packages/theme-doc/src/Layout/Footer.module.less

```diff
@@ -0,0 +1,30 @@
+@import 'rc-footer/assets/index.less';
+
+@footer-prefix-cls: footer;
```

Comment:
We can import a less file and overwrite variables in it like this.
Ref: https://stackoverflow.com/a/19514889